packages/server/graphql/public/typeDefs/NewMeeting.graphql

```diff
@@ -0,0 +1,121 @@
+"""
+A team meeting history for all previous meetings
+"""
+interface NewMeeting {
+  """
+  The unique meeting id. shortid.
+  """
+  id: ID!
+
+  """
+  The timestamp the meeting was created
+  """
+  createdAt: DateTime!
+
+  """
+  The id of the user that created the meeting
+  """
+  createdBy: ID!
+
+  """
+  The user that created the meeting
+  """
+  createdByUser: User!
+
+  """
+  The timestamp the meeting officially ended
+  """
+  endedAt: DateTime
+
+  """
+  The location of the facilitator in the meeting
+  """
+  facilitatorStageId: ID!
+
+  """
+  The userId (or anonymousId) of the most recent facilitator
+  """
+  facilitatorUserId: ID!
+
+  """
+  The facilitator team member
+  """
+  facilitator: TeamMember!
+
+  """
+  Is this locked for starter plans?
+  """
+  locked: Boolean!
+
+  """
+  The team members that were active during the time of the meeting
+  """
+  meetingMembers: [MeetingMember!]!
+
+  """
+  The auto-incrementing meeting number for the team
+  """
+  meetingNumber: Int!
+
+  """
+  The id of the meeting series this meeting belongs to
+  """
+  meetingSeriesId: ID
```

Comment:
A few fields were defined in code but not the `_legacy.graphql`. I've added these here: `locked`, `meetingSeriesId`, `summary` and `scheduledEndTime`.